security/restricting-fiddler-features.md

```diff
@@ -0,0 +1,76 @@
+
+## Restricting Features on macOS
+
+
+Users on macOS with administrative privileges set policies through the [**defaults**](https://macos-defaults.com/) command and by using the `com.progress-telerik.fiddler` domain alongside the following policies:
+
+| Policy          | Description | Value Type |  Value to apply the policy
+|:-----------------|:----------------|:----------------|:-----------------
+| **DisableCloud**   | Disables sharing options  | integer  | 1
+| **DefaultProxyPort**   | Sets default proxy port  | integer  | A number between 0 and 95535
+| **DisableProxyPortChange**   | Disables the change of port  | integer  | 1
+| **DefaultBypassList**   | Sets default bypass list  | string  | Add URLs with “;” in between
+| **DisableBypassListChange**   | Disables the change of the bypass list | integer  | 1
+| **DisableAllowRemoteDevicesToConnect**   | Disables option to connect to remote devices | integer  | 1
+| **DisableAnalytics**   | Disables Fiddler's analytics  | integer  | 1
+
+Setting a policy through the **defaults** command
+```sh
+defaults write com.progress-telerik.fiddler DisableAllowRemoteDevicesToConnect 1
+```
+
+For the **DefaultProxyPort** policy, set the value to be integer as follows:
+```sh
+defaults write com.progress-telerik.fiddler DefaultProxyPort -integer 8899
+```
+
+Reading the set policies for the Fiddler Everywhere application
+```sh
+defaults read com.progress-telerik.fiddler
+```
+
+Deleting set policies for the Fiddler Everywhere application
+```sh
+defaults delete com.progress-telerik.fiddler
+```
+
+## Restricting Features on Windows
+
+Users on Windows with administrative privileges set policies through the Windows registry editor by setting the following policies:
+
+| Policy          | Description | Regedit Value Type |  Value to apply the policy
+|:-----------------|:----------------|:----------------|:-----------------
+| **DisableCloud**   | Disables sharing options  | DWORD-32 (hexadecimal)  | 1
+| **DefaultProxyPort**   | Sets default proxy port  | DWORD-32 (decimal)  | A number between 0 and 95535
+| **DisableProxyPortChange**   | Disables the change of port  | DWORD-32 (hexadecimal)  | 1
+| **DefaultBypassList**   | Sets default bypass list  | String Value  | Add URLs with “;” in between
+| **DisableBypassListChange**   | Disables the change of the bypass list | DWORD-32 (hexadecimal)  | 1
+| **DisableAllowRemoteDevicesToConnect**   | Disables option to connect to remote devices | DWORD-32 (hexadecimal)  | 1
+| **DisableAnalytics**   | Disables Fiddler's analytics  | DWORD-32 (hexadecimal)  | 1
+
+Set the above policies as follows:
+
+1. Open the Windows Registry Editor.
+
+1. Navigate to **HKEY_CURRENT_USER\SOFTWARE\Policies**
```

Comment:
We should not explain how to set the registry. We must explain which are the registry keys that are respected by the application.
And please add information that the app supports both HKCU and HKLM and if a key is present in both places, HKLM's value will be used.